tasks/other/run-js-tests.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+[ -x "$(command -v npm)" ] || { >&2 echo "npm is required but not installed, exiting."; exit 1; }
+
+pushd /app/source/public/wp-content/plugins/planet4-plugin-gutenberg-blocks || exit
+npm test
```

Comment:
Instead of changing directory and then changing back, it's easier to run npm with `--prefix path/to/your/app`